Code/Editor/NewLevelDialog.h

```diff
@@ -56,12 +49,6 @@ protected slots:
 public:
     QString         m_level;
     QString         m_levelFolders;
-    bool                m_bIsResize;
-    bool                m_bUpdate;
-
-    std::vector<QString>    m_itemFolders;
-
     QScopedPointer<Ui::CNewLevelDialog> ui;
     bool m_initialized;
-};
-#endif // CRYINCLUDE_EDITOR_NEWLEVELDIALOG_H
+};
```

Comment:
The final  newline has been removed from this file.
```suggestion
};

```